packages/fluent-theme/src/icons/DialpadIcon.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { SVGAttributes } from 'react';
+
+export function DialpadIcon(props: SVGAttributes<SVGElement>) {
+  return (
+    <svg
+      aria-hidden="true"
+      fill="currentColor"
+      height="1em"
```

Comment:
Is it really 1em x 1em? It's not a square.

Reply:
That's how modern fluent renders icons. Why 1em width is any different from 1em height?

I'm going to switch to `font-size` for icons instead of `width/height: 20px`

![image](https://github.com/microsoft/BotFramework-WebChat/assets/2841858/4b02ccd5-58d5-4995-a404-6d6539cf205f)